src/Greenshot.Editor/Forms/ImageEditorForm.cs

```diff
@@ -1292,6 +1297,7 @@ private void BindFieldControls()
             new BidirectionalBinding(previewQualityUpDown, "Value", _surface.FieldAggregator.GetField(FieldType.PREVIEW_QUALITY), "Value",
                 DecimalDoublePercentageConverter.GetInstance(), NotNullValidator.GetInstance());
             new BidirectionalBinding(obfuscateModeButton, "SelectedTag", _surface.FieldAggregator.GetField(FieldType.PREPARED_FILTER_OBFUSCATE), "Value");
+            new BidirectionalBinding(cropModeButton, "SelectedTag", _surface.FieldAggregator.GetField(FieldType.CROPMODE), "Value");
```

Comment:
Note: just saying, as it's not your addition I can express my horror: the BidirectionalBinding thingies are horrible, I cannot stand a new without assigning them... but I didn't write that part, it's grown hysterically. (historically)